Classes/Utility/FLEXRuntimeUtility.m

```diff
@@ -277,14 +277,22 @@ + (NSArray *)prettyArgumentComponentsForMethod:(Method)method
     NSMutableArray *components = [NSMutableArray array];
     
     NSString *selectorName = NSStringFromSelector(method_getName(method));
-    NSArray *selectorComponents = [selectorName componentsSeparatedByString:@":"];
-    unsigned int numberOfArguments = method_getNumberOfArguments(method);
+    NSMutableArray *selectorComponents = [[selectorName componentsSeparatedByString:@":"] mutableCopy];
     
-    for (unsigned int argIndex = kFLEXNumberOfImplicitArgs; argIndex < numberOfArguments; argIndex++) {
-        char *argType = method_copyArgumentType(method, argIndex);
-        NSString *readableArgType = [self readableTypeForEncoding:@(argType)];
+    // this is a workaround cause method_getNumberOfArguments() returns wrong number for some methods
+    if (selectorComponents.count == 1) {
+        return [selectorComponents copy];
+    }
```

Comment:
This doesn't seem quite right. If a selector has one component, then it doesn't take any arguments. This change causes [every method screen to have a non-existent argument field.](https://i.imgur.com/W2CqMVw.png)

I think we should revert this in a new commit.

Reply:
@tikoyesayan @ryanolsonk